### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by asolntsev**
just fixes some typos in TesTNG code/tests/comments.

### CODE REVIEW

File: testng-core/src/main/java/org/testng/internal/invokers/TestMethodWorker.java [hunk 1]
```diff
@@ -109,32 +109,32 @@ public String toString() {
...
-        && doesTaskHavePreRequistes()
+        && doesTaskHavePreRequisites()
         && currentThreadId != threadIdToRunOn) {
       completed = false;
       return;
     }
 
-    for (IMethodInstance testMthdInst : m_methodInstances) {
-      ITestNGMethod testMethod = testMthdInst.getMethod();
+    for (IMethodInstance testMethdInstance : m_methodInstances) {
```
Comment (juherr) [inline testng-core/src/main/java/org/testng/internal/invokers/TestMethodWorker.java:118]: I think you can add the last missing 'o' 😉

Comment (asolntsev) [reply testng-core/src/main/java/org/testng/internal/invokers/TestMethodWorker.java]: Oops :) 
Fixed.

**Review by juherr — APPROVED** [review]

**Review by krmahadevan — APPROVED** [review]

**Review by krmahadevan — CHANGES_REQUESTED** [review]
The build is failing because of formatting issues.
Can you please do the following ?

1. `./gradlew autostyleApply` (This will apply code formatting)
2. Commit the changes.
3. Run `./gradlew clean check` (This will ensure that the code formatting is checked and all the tests are run)
4. Push the changes after the tests have passed so that the PR gets updated.

**Review by krmahadevan — APPROVED** [review]

**Comment by krmahadevan** [comment]
@asolntsev - Thanks for the PR. This now is merged. Look forward to more contributions from you

**Comment by asolntsev** [comment]
@krmahadevan I see the project doesn't compile on "17, Adopt OpenJ9":

```
SimpleBaseTest.kt: (402, 24): Unresolved reference: dumpHeap
```

Apparently, this JVM doesn't have method `HotSpotDiagnosticMXBean.dumpHeap()`.
Shouldn't we remove this JVM from build matrix?

**Comment by krmahadevan** [comment]
Or we would need to find a way to ensure that this test doesn't run on this JVM. We need to take a closer look on this. Thanks for highlighting this.

**Pull request merged**